README.md

```diff
@@ -35,7 +35,7 @@ This repository contains examples of various algorithms written on different pro
 | [Queue](https://en.wikipedia.org/wiki/Queue_(abstract_data_type))                               |                                       | [:octocat:](queue/Cpp)                |                                       |                                       |
 | [Stack](https://en.wikipedia.org/wiki/Stack_(abstract_data_type))                               | [:octocat:](stack/C )                 |                                       | [:octocat:](stack/Java)               | [:octocat:](stack/Python)             |
 | [Linear Linked List](https://en.wikipedia.org/wiki/Linked_list)                                 | [:octocat:](linked_list/C)            | [:octocat:](linked_list/Cpp)          |                                       | [:octocat:](linked_list/Python)       |
-| [AVL Tree](https://en.wikipedia.org/wiki/AVL_tree)                                              | [:octocat:](avl_tree/C)               |                                       | [:octocat:](avl_tree/Java)            | [:octocat:](avl_tree/Python)          |
+| [AVL Tree](https://en.wikipedia.org/wiki/AVL_tree)                                              | [:octocat:](avl_tree/C)               | [:octocat:](avl_tree/Cpp)                          | [:octocat:](avl_tree/Java)            | [:octocat:](avl_tree/Python)          |
```

Comment:
Cpp column characters should be 40. It is more than 40 which is making it little odd.